src/Features/Core/Portable/AddImport/References/SymbolReference.cs

```diff
@@ -87,19 +88,31 @@ public override int GetHashCode()
                     return null;
                 }
 
-                if (!this.SearchResult.DesiredNameMatchesSourceName(document))
+                var isFuzzy = !this.SearchResult.DesiredNameMatchesSourceName(document);
+                var tags = GetTags(document);
+                if (isFuzzy)
                 {
-                    // The name is going to change.  Make it clear in the description that 
-                    // this is going to happen.
+                    // The name is going to change.  Make it clear in the description that this is
+                    // going to happen.
                     description = $"{this.SearchResult.DesiredName} - {description}";
+
+                    // if we were a fuzzy match, and we didn't have any glyph to show, then add the
+                    // namespace-glyph to this item. This helps indicate that not only are we fixing
+                    // the spelling of this name we are *also* adding a namespace.  This helps as we
+                    // have gotten feedback in the past that the 'using/import' addition was
+                    // unexpected.
+                    if (tags.IsDefaultOrEmpty)
```

Comment:
so, icon is not composite, but shows more important concept<?> I guess? all fix is adding import I guess? but if one add more than import like adding reference, then it shows that glyph?